services/api/src/routes/auth.js

```diff
@@ -16,12 +16,10 @@ const passwordField = Joi.string()
 router
   .post(
     '/register',
-    validate({
-      body: Joi.object({
-        email: Joi.string().lowercase().email().required(),
-        name: Joi.string().required(),
-        password: passwordField.required(),
-      }),
+    validateBody({
```

Comment:
Just confirming It it still possible todo

`validateBody(Joi.object({}).advancedFeature())`

I think that would be very useful for those rare but super useful times when you want to use xor/oneOff or other complex schemas 

You write
```allowed both helpers to accept either a Joi schema or plain object```
Do i understand that correctly?

Reply:
ya basically it's this:

```
const schema = Joi.isSchema(obj) ? obj : Joi.object(obj);
```

Reply:
also just to confirm the behavior here is exactly as before just split out, so yes it does set `ctx.request.body` and do some other special stuff for the query (details are still there in comments) which it needs to do as the validator can modify the input object